src/three/math/OBB.js

```diff
@@ -17,25 +24,42 @@ export class OBB {
 		inverseTransform.copy( transform ).invert();
 
 		const { min, max } = box;
-		let index = 0;
-		for ( let x = - 1; x <= 1; x += 2 ) {
+		points[ 0 ].set( min.x, min.y, min.z ).applyMatrix4( transform ); // Front-bottom-left
+		points[ 1 ].set( max.x, min.y, min.z ).applyMatrix4( transform ); // Front-bottom-right
+		points[ 2 ].set( max.x, max.y, min.z ).applyMatrix4( transform ); // Front-top-right
+		points[ 3 ].set( min.x, max.y, min.z ).applyMatrix4( transform ); // Front-top-left
 
-			for ( let y = - 1; y <= 1; y += 2 ) {
+		points[ 4 ].set( min.x, min.y, max.z ).applyMatrix4( transform ); // Back-bottom-left
+		points[ 5 ].set( max.x, min.y, max.z ).applyMatrix4( transform ); // Back-bottom-right
+		points[ 6 ].set( max.x, max.y, max.z ).applyMatrix4( transform ); // Back-top-right
+		points[ 7 ].set( min.x, max.y, max.z ).applyMatrix4( transform ); // Back-top-left
+		this.planeNeedsUpdate = true;
```

Comment:
I don't think there's any reason to defer this plane generation, right? We can just update them here after the points in the "update" function.

Reply:
The idea was often / sometimes, the planes are not needed as intersectsFrustum can return early before reaching the planes check.
I didn't test for any tradeoff or if neglectable. 
I'll go with the idea you prefer.